tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -8,6 +8,7 @@ import (
 	"encoding/base64"
 	"encoding/pem"
 	"fmt"
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
it feels a little weird importing an azure library into the aws wni. But oh well

Reply:
Thanks for catching this. I did not realize this. I will revert the change.